src/base.jl

```diff
@@ -829,33 +829,45 @@ function IS.add_time_series_from_file_metadata_internal!(
     cache::IS.TimeSeriesParsingCache,
     file_metadata::IS.TimeSeriesFileMetadata,
 )
+    associations = TimeSeriesAssociation[]
     IS.set_component!(file_metadata, data, PowerSystems)
     component = file_metadata.component
     if isnothing(component)
-        return
+        return associations
     end
 
     ts = IS.make_time_series!(cache, file_metadata)
     if component isa AggregationTopology && file_metadata.scaling_factor_multiplier in
        ["get_max_active_power", "get_max_reactive_power"]
-        uuids = Set{UUIDs.UUID}()
+        uuids = Set{Base.UUID}()
         for bus in _get_buses(data, component)
             push!(uuids, IS.get_uuid(bus))
         end
         for _component in (
             load for load in IS.get_components(ElectricLoad, data) if
             IS.get_uuid(get_bus(load)) in uuids
         )
-            IS.add_time_series!(data, _component, ts; skip_if_present = true)
+            file_metadata.component = _component
+            if !IS.has_assignment(cache, file_metadata)
+                IS.add_assignment!(cache, file_metadata)
+                push!(associations, TimeSeriesAssociation(_component, ts))
+            end
+        end
+        file_metadata.component = component
+        orig_sf = file_metadata.scaling_factor_multiplier
+        try
+            file_metadata.scaling_factor_multiplier = replace(orig_sf, "max" => "peak")
+            area_ts = IS.make_time_series!(cache, file_metadata)
+            IS.add_assignment!(cache, file_metadata)
+            push!(associations, TimeSeriesAssociation(component, area_ts))
+        finally
+            file_metadata.scaling_factor_multiplier = orig_sf
         end
-        file_metadata.scaling_factor_multiplier =
-            replace(file_metadata.scaling_factor_multiplier, "max" => "peak")
-        area_ts = IS.make_time_series!(cache, file_metadata)
-        key = IS.add_time_series!(data, component, area_ts; skip_if_present = true)
```

Comment:
@claytonpbarrows Can you review this change? I flipped the logic around checking for duplicates. Instead of `skip_if_present`, the parsing process tracks the time series that have been added and then skips duplicates. That simplified the time series management code in IS. This line was setting skip = true for Area/LoadZone. I don't think that was correct. Can you confirm?

Also, the PSY tests don't use this feature. Can you point me to a system file (time_series_pointers.json) that does use it?